jobs/setup.py

```diff
@@ -35,7 +35,7 @@ def read_file(filename):
 
 
 setup(name='buildhub',
-      version='1.1.5.dev0',
+      version='1.1.5',
```

Comment:
I get nervous around version stuff. Is `1.1.5 > 1.1.5.dev0`?

Reply:
Yes. BTW we use `zest.releaser` in most of our python projects.